src/screens/ParadeMapScreen/Map.js

```diff
@@ -37,12 +41,19 @@ const shouldNeverAsk = (status: PermissionStatus) => status === "restricted";
 type Props = {
   route: Array<Coordinates>,
   paradeRegion: Region,
-  terminals: Array<Terminals>
+  terminals: Array<Terminals>,
+  stages: Array<Event>,
+  savedEvents: SavedEvents,
+  addSavedEvent: string => void,
+  removeSavedEvent: string => void,
+  onPress: (id: string) => void
```

Comment:
A better name for this might be `onStagePress`. A simple `onPress` on a `Map` component suggests that is called when the map itself is pressed.